crates/eframe/src/native/run.rs

```diff
@@ -23,6 +23,7 @@ pub use epi::NativeOptions;
 enum EventResult {
     Wait,
     RepaintAsap,
+    RepaintNext,
```

Comment:
> there was no comment explaining it so I am unsure.

...and you continue this bad practice by does not including comment that describe the difference from `RepaintAsap` :smile:

Reply:
...whoops. :D

I shall write some doc comments about it, commit incoming soon.